language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/duplication_tests.rs

```diff
@@ -9,6 +9,8 @@ proptest! {
     #[test]
     fn valid_duplication(module in CompiledModule::valid_strategy(20)) {
         let duplication_checker = DuplicationChecker::new(&module);
-        prop_assert!(duplication_checker.verify().is_ok());
+        let res = duplication_checker.verify();
+        println!("{:?}", res);
```

Comment:
the `println` should probably go, right? in fact we could revert this, can we?

Reply:
Ah my bad, forgot to clean this up. Will revert before landing.